saleor/payment/gateways/adyen/plugin.py

```diff
@@ -94,7 +94,7 @@ class AdyenGatewayPlugin(BasePlugin):
             ),
             "label": "Client Key",
         },
-        "Origin Url": {
+        "origin-url": {
             "type": ConfigurationTypeField.STRING,
             "help_text": (
                 "The origin URL of the page where you are rendering the Drop-in. This "
```

Comment:
What's a Drop-in? Should this link to Adyen docs?

Reply:
Added url